style.css

```diff
@@ -107,9 +107,373 @@
   display: inline-block;
   padding-left: 24px;
   padding-top: 60px;
 }
+
+.container {
+  display: grid;
+  background-color: #e5e5e5;
+  height: 2900px;
+  width: 375px;
+  grid-template-rows: 1fr, 1fr, 1fr, 1fr;
+  row-gap: 12px;
+  align-content: center;
+  justify-content: center;
+  margin-top: 20px;
+}
+
+.items {
+  background-color: #fff;
+  width: 327px;
+  height: 552px;
+  margin-top: 114px;
+  border-radius: 16px;
+  padding-left: 16px;
+}
+
+.workimg {
+  margin-top: 16px;
+}
+
+.items h1 {
+  font-family: 'Poppins', sans-serif;
+  font-style: bold;
+  font-size: 32px;
+  color: #172b4d;
+  line-height: 44px;
+  line-height: 92%;
+  padding-top: 12px;
+  text-align: left;
+  vertical-align: top;
+  width: 295px;
+  height: 36px;
+  font-weight: 700px;
+}
+
+.items h2 {
+  font-family: 'Poppins', sans-serif;
+  font-style: bold;
+  font-size: 13px;
+  color: #344563;
+  line-height: 16px;
+  line-height: 82%;
+  padding-top: 20px;
+  text-align: left;
+  vertical-align: top;
+  font-weight: 600;
+  width: 45px;
+  height: 16px;
+}
+
+.work-ul {
+  display: flex;
+  list-style: none;
+}
+
+.work ul h3 {
+  display: inline-block;
+  height: 16px;
+  line-height: 16px;
+  padding-left: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 600;
+  font-size: 13px;
+  font-style: semibold;
+  padding-top: 16px;
+  color: #7a869a;
+}
+
+.items p {
+  padding-top: 20px;
+  width: 295px;
+  height: 76px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  font-style: regular;
+  text-align: left;
+  vertical-align: top;
+  color: #344563;
+}
+
+.work-tags {
+  display: flex;
+  list-style: none;
+  padding-top: 16px;
+  flex-wrap: wrap;
+}
+
+.work-tags .html {
+  width: 53px;
+  height: 24px;
+  background-color: #ebebff;
+  text-align: center;
+  margin-top: 12px;
+  border-radius: 8px;
+  font-family: 'Poppins', sans-serif;
+  margin-right: 8px;
+  color: #6070ff;
+  font-style: medium;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 16px;
+  letter-spacing: 3%;
+  padding-top: 4px;
+}
+
+.work-tags .css {
+  width: 45px;
+  height: 24px;
+  background-color: #ebebff;
+  text-align: center;
+  margin-top: 12px;
+  border-radius: 8px;
+  font-family: 'Poppins', sans-serif;
+  margin-right: 8px;
+  color: #6070ff;
+  font-style: medium;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 16px;
+  letter-spacing: 3%;
+  padding-top: 4px;
+}
+
+.work-tags .javascript {
+  width: 89px;
+  height: 24px;
+  background-color: #ebebff;
+  text-align: center;
+  margin-top: 12px;
+  border-radius: 8px;
+  font-family: 'Poppins', sans-serif;
+  margin-right: 8px;
+  color: #6070ff;
+  font-style: medium;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 16px;
+  letter-spacing: 3%;
+  padding-top: 4px;
+}
+
+.button {
+  width: 124px;
+  height: 48px;
+  margin-top: 20px;
+  border-radius: 8px;
+  padding: 8px;
+  text-align: center;
+  cursor: pointer;
+}
```

Comment:
Your buttons look great. Kindly ensure that you include the button interactions too as shown on the figma design.

Your current hover interaction:
![image](https://user-images.githubusercontent.com/20554583/212428144-622235c2-57ac-4868-84ef-83fdfd7ca450.png)

Figma button interactions:
![image](https://user-images.githubusercontent.com/20554583/212428223-362aa2a9-2a76-4871-9d96-25a0949fc389.png)